tests/php/test-amp-analytics-options.php

```diff
@@ -240,12 +240,16 @@ public function test_amp_get_analytics() {
 		add_filter(
 			'amp_analytics_entries',
 			static function( $analytics ) use ( $key ) {
-				$analytics[ $key ]['type'] = 'test';
+				$analytics[ $key ]['type']                                = 'test';
+				$analytics[ $key ]['attributes']['data-include']          = '_till_responded';
+				$analytics[ $key ]['attributes']['data-block-on-consent'] = 'credentials';
 				return $analytics;
 			}
 		);
 		$analytics = amp_get_analytics();
 		$this->assertEquals( 'test', $analytics[ $key ]['type'] );
+		$this->assertEquals( '_till_responded', $analytics[ $key ]['data-include'] );
+		$this->assertEquals( 'credentials', $analytics[ $key ]['data-block-on-consent'] );
```

Comment:
This should be `$analytics[ $key ]['attributes']['data-block-on-consent']` it seems?

Reply:
@swissspidy Thanks for these. I believe I've resolved the issues. Unable to get the phpunit tests to run on my machine. Getting:

```
PHP Fatal error:  Unable to locate wordpress-tests-lib in C:\Users\ethan\dev-archive\amp-wp\vendor\xwp\wp-dev-lib\sample-config\phpunit-plugin-bootstrap.php on line 40
```

And that's running `phpunit` directly rather than `npm run test:php` which can't find `phpunit`, even though it's executable on my path.